tree-sitter-dscexpression/corpus/invalid_expressions.txt

```diff
@@ -110,3 +110,17 @@ Incomplete expression
         (function
           (functionName)))
       (MISSING "]"))
+
+=====
+Invalid float argument
+=====
+[myFunction(a.1)]
```

Comment:
Added tests. To clarify what I said before - `+1` is unsupported by the grammar. `.1` is supported by the grammar, but detected as float within the Rust parsing, so unsupported overall.

Reply:
The Rust parsing _could_ support floats - it's just not setup to right now, whether it's `0.1` or `.1`.

I'm having trouble trying to exclude leading with the decimal point in the grammar because then the grammar detects the decimal portion as the number, which results in the functions like `int()` seemingly handling the input incorrectly -perhaps the Rust parser needs to check for any error nodes and stop execution if it finds one?

Reply:
See my new comment in the regex below